packages/now-node/test/fixtures/12-stack-trace-esm/index.js

```diff
@@ -0,0 +1,10 @@
+export default function handler(req, res) {
+  try {
+    if (req) {
+      throw new Error('Should throw');
```

Comment:
```suggestion
      req.notdefined.something;
```

Let's use the same code from #645 to prove it is fixed.